tensorboard/plugins/image/tf_image_dashboard/tf-image-dashboard.html

```diff
@@ -139,6 +153,23 @@ <h3>No image data was found.</h3>
     </tf-dashboard-layout>
     <style include="dashboard-style"></style>
     <style>
+      .resettable-slider-container {
+        display: flex;
+      }
+      .resettable-slider-container paper-slider {
+        flex-grow: 1;
+      }
+      .resettable-slider-container paper-button {
+        flex-grow: 0;
+      }
+      paper-button[disabled] {
```

Comment:
Could we confine this to only `paper-button`s within `resettable-slider-container` like the other specifications?